lean/commands/data/generate.py

```diff
@@ -106,13 +126,24 @@ def generate(start: datetime,
     lean_config_manager = container.lean_config_manager
     data_dir = lean_config_manager.get_data_directory()
 
+    if tickers:
+        tickers_options = ["--tickers=" + ",".join(tickers)]
+    else:
+        tickers_options = []
+
+    if tickers and symbol_count != len(tickers):
+        raise BadParameter(
+                f"--symbol-count ({symbol_count}) != number of tickers ({len(tickers)})"
```

Comment:
Think this should be enforced by the data generator if it needs too? We can remove it from here 👍